core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.mainpage.html

```diff
@@ -1,26 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Topics and Skills Dashboard - Oppia"})
-  </head>
-  <body>
-    <oppia-root>
-      <div ng-controller="Base">
-        <oppia-base-content>
-          <navbar-breadcrumb>
-            <oppia-topics-and-skills-dashboard-navbar-breadcrumb>
-            </oppia-topics-and-skills-dashboard-navbar-breadcrumb>
-          </navbar-breadcrumb>
-
-          <content>
-            <oppia-topics-and-skills-dashboard-page></oppia-topics-and-skills-dashboard-page>
-          </content>
-          <page-footer></page-footer>
-        </oppia-base-content>
-      </div>
-
-    </oppia-root>
-    @load('pages/footer_js_libs.html')
-    <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=default"></script>
```

Comment:
Don't we need to do something for MathJax? (Are you able to use MathJax on the dashboard page?)

Reply:
Added, Done!!

Reply:
Hm, actually could you check if you're able to use Math formulae in the rich text field while creating a new skill via the topics dashboard? We should make sure that this still works.

Reply:
Yeah, I missed it, it was not working properly!!